File: atcoder/other/apg4b/c/apg4b-ex7.c
#include <stdio.h>
#include <stdbool.h>

int main(void) {
  // 変数a,b,cにtrueまたはfalseを代入してAtCoderと出力されるようにする。
  bool a = true;// true または false
  bool b = false;// true または false
  bool c = true;// true または false

  // ここから先は変更しないこと

  if (a) {
    printf("At");
  }
  else {
    printf("Yo");
  }

  if (!a && b) {
    printf("Bo");
  }
  else if (!b || c) {
    printf("Co");
  }

  if (a && b && c) {
    printf("foo!");
  }
  else if (true && false) {
    printf("yeah!");
  }
  else if (!a || c) {
    printf("der");
  }

  printf("\n");
}

File: atcoder/other/apg4b/python/apg4b-ex7.py
a=True
b=False
c=True

if a==True:
    print("At",end='')
else:
    print("Yo",end='')

if a == False and b == True:
    print("Bo",end='')
elif b == False or c == True:
    print("Co",end='')

if a == True and b == True and c == True:
    print("foo!")
elif True and False:
    print("yeah!")
elif a == False or c ==True:
    print("der")